packages/teleport/src/cluster/components/Support/Support.tsx

```diff
@@ -0,0 +1,199 @@
+            />
+            <Link title="Send Product Feedback" url="" />
+          </ul>
+        </MiniCard>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Resources
+          </h4>
+          <ul>
+            <Link
+              title="Quickstart Guide"
+              url="https://gravitational.com/teleport/docs/quickstart/"
+            />
+            <Link
+              title="tsh User Guide"
+              url="https://gravitational.com/teleport/docs/user-manual/"
+            />
+            <Link
+              title="Admin Guide"
+              url="https://gravitational.com/teleport/docs/admin-guide/"
+            />
+            <Link
+              title="Download Page"
+              url="https://gravitational.com/teleport/download"
+            />
+            <Link
+              title="FAQ"
+              url="https://gravitational.com/teleport/docs/faq/"
+            />
+          </ul>
+        </MiniCard>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Troubleshooting
+          </h4>
+          <ul>
+            <Link title="Monitoring Teleport" url="" />
+            <Link title="Collecting Debug Data" url="" />
+            <Link title="Troubleshooting FAQ" url="" />
+          </ul>
+        </MiniCard>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Updates
+          </h4>
+          <ul>
+            <Link title="Product Changelog" url="" />
+            <Link
+              title="Gravitational Blog"
+              url="https://gravitational.com/blog/"
+            />
+          </ul>
+        </MiniCard>
+      </CardWrapper>
+      <ClusterInfo>
+        <h4>Cluster Information</h4>
+        <ul>
+          <ClusterData title="Cluster Name" data={cluster.clusterId} />
+          <ClusterData title="Teleport Version" data={cluster.authVersion} />
+          <ClusterData title="Public Address" data={cluster.publicURL} />
+          <ClusterData title="Connected Nodes" data={cluster.nodeCount} />
+        </ul>
+      </ClusterInfo>
+    </SupportBox>
+  );
+};
+
+const Link = ({ title = '', url = '' }) => (
+  <li>
+    <a href={url} target="_blank">
+      {title}
+    </a>
+  </li>
+);
+
+const ClusterData = ({ title = '', data }) => (
+  <li>
+    <span>{title}: </span>
+    {data}
+  </li>
+);
+
+const SupportBox = styled(FeatureBox)`
+  h4 {
+    text-transform: uppercase;
+  }
+
+  & > div:not(:first-child) {
+    padding: 6px 30px 12px 30px;
+  }
+
+  ul {
+    padding-left: 0;
+    list-style-type: none;
+  }
+
+  li {
+    margin-bottom: 15px;
+  }
+`;
+
+const CardWrapper = styled(Card)`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+
+const MiniCard = styled.div`
+  width: 210px;
+
+  .icon {
+    font-size: 26px;
+    padding-right: 8px;
+    padding-bottom: 2px;
+  }
+
+  h4 {
+    border-bottom: 1px solid ${theme.colors.primary.dark};
+    line-height: 40px;
```

Comment:
Using `line-height: 40px;` to center a text is anti-pattern.

Reply:
removed